sdk/src/Core/Amazon.Runtime/AmazonServiceClient.cs

```diff
@@ -411,6 +416,9 @@ protected virtual void Dispose(bool disposing)
                 if (RuntimePipeline != null)
                     RuntimePipeline.Dispose();
 
+                if(_uptimeMetricMeasurer != null)
+                    _uptimeMetricMeasurer.Dispose();
+
```

Comment:
Should set the variable to NULL after dispose: https://learn.microsoft.com/en-us/dotnet/standard/garbage-collection/implementing-dispose#the-disposebool-method-overload . Also a simplified way is:

```
                _safeHandle?.Dispose();
                _safeHandle = null;

```